:pencil: make reference to other codes consistent (#40)
geomechanics/mpm: README.md

# High-Performance Material Point Method

> > Geomechanics MPM is the codebase utilized by the [Soga Research Group](http://geomechanics.berkeley.edu/) at the University of California, Berkeley. This code represents a fork of [CB-Geo](https://github.com/cb-geo/mpm). 

Reviewer: How about this?
```suggestion
> Geomechanics MPM is the codebase utilized by the [Soga Research Group](http://geomechanics.berkeley.edu/) at the University of California, Berkeley which is based on the [CB-Geo](https://github.com/cb-geo/mpm) framework. 
```
